README.md

```diff
@@ -1,9 +1,10 @@
-# loopback-example-access-control
+# loopback-example-access-control(use datasource cloudant)
 
 ```
 $ git clone https://github.com/strongloop/loopback-example-access-control
 $ cd loopback-example-access-control
 $ npm install
+Manually change credential of `mycloudant` in server/datasources.json to your cloudant credential
```

Comment:
Suggested change (i don't think we need the word "manually" :) ): 
Go to server/datasources.json and change the credential of `mycloudant` to your cloudant credential.